QgisModelBaker/gui/export.py

```diff
@@ -205,6 +213,25 @@ def refreshed_export_models_model(self):
 
         self.export_models_model.refresh_models(db_connector)
 
+    def db_ili_version(self, configuration):
```

Comment:
Looks like we can send this common method to a common place. I'll check if I can see where.

Reply:
Yes. I don't like to have it three times neither. But I didn't found a better place since it should be still in the GUI part using the configuration. So there could be a new file like `gui_utils.py` but in the end I thought it's not important enough to introduce a new file. 
Did you found a good place? Or should we leave it where it is and introduce a new file, when the common functions gets more than this?